### DISCUSSION TITLE
place window in multi-monitor setup based on mouse location

### DISCUSSION THREAD

**Discussion by theoephraim** [rxhanson/Rectangle#858] [Ideas]
Just a feature I miss from Divvy.

Would like the option to place the window on a specific monitor (in multi-monitor setup) based on mouse location. For example, if I have 2 monitors A and B, if I move the mouse onto screen B and hit the "left half" shortcut, the window gets placed on the left half of monitor B.

Of course there is the option to press a shortcut repeatedly to cycle through monitors, but that doesn't seem to work for the full screen shortcut.

**Comment by rxhanson** [comment]
Makes sense to me. I don't quite have the bandwidth to add it in myself at the moment but would definitely welcome a good pull request for it.
